end_to_end/tpu/mixtral/8x7b/1_test_mixtral.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This file, combined with step 2 in the same directory, demonstrates:
```

Comment:
Perhaps mention that we run this test nightly

https://github.com/google/maxtext/blob/db31dd4b0b686bca4cd7cf940917ec372faa183a/end_to_end/tpu/gemma/7b/1_test_gemma.sh#L3C1-L3C2

Reply:
Sure, will add nightly run.

Reply:
Added `runs on daily basis`